CHANGES.rst

```diff
@@ -33,6 +33,8 @@ cube_build
 
 - Replace scale1 and scale2 arguments with scalexy, add debug option debug_spaxel,
   and add more details to docs. [#7783]
+- Fix output WCS for IFU cubes built in internal_cal coordinates, for NIRSpec
+  calibration analysis. [#7811]
```

Comment:
```suggestion

- Fix output WCS for IFU cubes built in internal_cal coordinates, for NIRSpec
  calibration analysis. [#7811]
```

Reply:
Can you elaborate a bit on exactly how the output WCS has been fixed? what was wrong with it before?

Reply:
I just pushed a clarification: the corrections were to the output scale and orientation for slicer coordinates.